build/config/pmm-agent.service

```diff
@@ -6,6 +6,8 @@ After=time-sync.target network.target
 Type=simple
 PermissionsStartOnly=true
 ExecStart=/usr/sbin/pmm-agent --config-file=/usr/local/percona/pmm-agent.yaml
+Restart=on-failure
+RestartPreventExitStatus=1
```

Comment:
Why? We should still restart it even with exit status 1.

I think we should also add
```
RestartSec=2s
TimeoutStopSec=10s
```